testing_exercise/testing.js

```diff
@@ -0,0 +1,62 @@
+function replaceWith(string, ltrOld, ltrNew) {
+	var newString = "";
+	for(var i=0; i<string.length; i++) {
+		if(string[i]==ltrOld) {
+			newString += ltrNew;
+		} else {
+			newString += string[i];
+		}
+	};
+	if(newString === "") {
+		return string;
+	};
+	return newString;
+}
+
+//
+
+function expand(arr, num) {
+	var newArr = [];
+	for(var i=0; i < arr.length*num; i++) {
+		newArr.push(arr[i % arr.length]);
+	}
+	return newArr;
+}
+
+//
+
+function acceptNumbersOnly(args) {
+	for(var i=0; i<arguments.length; i++) {
+		if(isNaN(arguments[i]))
```

Comment:
This only checks to see if it is `NaN`, what about other data types? You might want to structure the logic so that you check to see **if** it is a number instead of if it is `NaN`